README.md

```diff
@@ -1,282 +1,257 @@
-# Vault Wallet React Native Android Module
+# Vault Engine SDK Docuemtation
 
-Android Native Module wrapper for React Native clients. Customers can use vault wallet as a library to enable access to blockchain assets, key management, backup, contacts, price feed, and eventually smart contracts. 
+By integrating with Vault Engine, devlopers can effecivly introduce a plug and play backend to your application, largly side stepping responsibility of devloping, managing, and maintaining many of the essential backend features critical to blockchain dapp devlopment. A more percise overview of features is avaliable and why you should consider Vault Engine for you DAPP checkout vaultwallet.io/VaultEngineSDK
 
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+These instructions will get you give you the resources needed to integrate Vault Engine into your product design roadmap, as well as begin devlopment. The below SDK documentation is to be taken as a guide to devloping your application with the Vault Engine SDK. Because the SDK is writen in kotlin nativly, the specific language you are using may present some differences in coding structure (callbacks etc). To simplify and ease the learning curve, the below documentation is writen to be readable and funtional, not exact code snippets. 
```

Comment:
remove 'kotlin' for now and just say 'native to the target platform'

Reply:
and say in the future we will add a way to test the api on the website itself with code